documentation/src/docs/table-api.md

```diff
@@ -410,16 +450,19 @@ tables:
     format:
       type: json
       fail-on-missing-field: true
-      derive-schema: true
     schema:
       - name: category
-        type: VARCHAR
+        data-type: STRING
       - name: value
-        type: INT
+        data-type: INT
+      - name: timestamp
+        data-type: TIMESTAMP(3)
 
 functions: [] 
 
 execution:
+  # either 'old' (default) or 'blink' , blink planner is recommended.

```

Comment:
Add the link https://ci.apache.org/projects/flink/flink-docs-release-1.10/dev/table/common.html#main-differences-between-the-two-planners for information.